src/Dialect/ONNX/ShapeInference/ArgMin.cpp

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
This file looks identical to ArgMax, anything we can do about that?

Reply:
Good point. I can create a template.